src/classes/com/sun/tdk/jcov/instrument/asm/ClassMorph.java

```diff
@@ -414,6 +414,7 @@ public static long computeCheckSum(byte[] classfileBuffer) {
                 case 16: // methodtype
                     i += 2;
                     break;
+                case 17: // dynamic
```

Comment:
So this is the real fix, everything else is cosmetic, am I reading it correctly?